docs/learn/avalanche/subnets-overview.md

```diff
@@ -2,107 +2,92 @@
 sidebar_label: Subnets
 ---
 
-# Subnets Overview
+# What Is a Subnet?
 
 A **Subnet** is a sovereign network which defines its own rules regarding its
 membership and token economics. It is composed of a dynamic subset of Avalanche
 validators working together to achieve consensus on the state of one or more
-blockchains. Each blockchain is validated by exactly one Subnet, and a Subnet
-can have many blockchains. A validator may be a member of many Subnets.
+blockchains. Each blockchain is validated by exactly one Subnet, while a Subnet
+can host many blockchains.
 
-Avalanche's 3 built-in blockchains: Platform Chain (P-Chain), Contract Chain
-(C-Chain) and Exchange Chain (X-Chain) are validated and secured by all the
-Avalanche validators which compose a special Subnet and is referred as the
-Primary Network.
+Avalanche's [Primary Network](avalanche-platform.md) is a special Subnet running three blockchains:
+
+- The Platform Chain (P-Chain)
```

Comment:
Link out to the P-Chain, C-Chain and X-Chain details 

- The Platform Chain [(P-Chain)](./avalanche-platform.md#platform-chain-p-chain)
- The Contract Chain [(C-Chain)](./avalanche-platform.md#contract-chain-c-chain)
- The Exchange Chain [(X-Chain)](./avalanche-platform.md#exchange-chain-x-chain)